src/api/spec/controllers/webui/configuration_controller_spec.rb

```diff
@@ -0,0 +1,100 @@
+require 'rails_helper'
+
+RSpec.describe Webui::ConfigurationController do
+  let(:confirmed_user) { create(:confirmed_user) }
+  let(:admin_user) { create(:admin_user) }
+
+  describe 'GET #index' do
```

Comment:
I don't like this. We should test that this controller has `before_filter :require_admin` and that Webui::WebuiController#require_admin works